src/components/item/sharing/ResendInvitationRenderer.tsx

```diff
@@ -30,7 +31,12 @@ const ResendInvitationRenderer = (
     };
 
     return (
-      <Button variant="outlined" onClick={resendEmail} disabled={clicked}>
+      <Button
+        variant="outlined"
+        onClick={resendEmail}
+        disabled={clicked}
+        className={ITEM_RESEND_INVITATION_BUTTON_CLASS}
```

Comment:
Out of curiosity is there a reason why you prefered using a `class` selector over an `id` selector ?

Reply:
Mostly because I didn't figure out how to send the correct id as a prop, and since I only was interested to see if there was a button or not. Also because a class selector was used for the permissions selector.